src/main/groovy/com/github/benmanes/gradle/versions/reporter/PlainTextReporter.groovy

```diff
@@ -36,6 +37,8 @@ class PlainTextReporter extends AbstractReporter {
   def write(printStream, Result result) {
     writeHeader(printStream)
 
+    writeGradleUpdates(printStream, result)
```

Comment:
I think we should push this to the bottom, after dependencies instead of before. For projects with a lot of dependencies, this info will be lost as they watch the bottom half for things to update. So putting it last makes it standout, since upgrading the build tool is valuable.